README.md

```diff
@@ -1,2 +1,93 @@
 # rotary-utils
 Circularization-related utilities for the rotary pipeline
+
+## Overview
+*rotary-utils*, currently under development, is a suite of utilities for manipulating circular DNA elements.
+Once completed, these utilities will be able to serve as a modern replacement for
+[circlator](https://github.com/sanger-pathogens/circlator), which is now in a frozen development state.
+The utilities in *rotary-utils* are built into the [*rotary* project](https://github.com/rotary-genomics/rotary),
```

Comment:
@LeeBergstrand Thanks, good point. I had originally wanted to use `rotary-utils`, but the hyphen was not acceptable for the `rotary_utils` folder, so I changed the hyphen to an underscore for folder and file names. You are right that we should make things consistent and also follow best practices.

Best practices seem to be covered in [PEP 8](https://peps.python.org/pep-0008/#package-and-module-names). Package names should ideally be all lowercase. Underscores are allowed but are discouraged. Modules can have underscores if needed, otherwise lowercase letters should be used. From some additional reading, it seems that distribution names have more flexibility, e.g., `sklearn` (package) is distributed as `scikit-learn'.

Based on this reading, I think it might be better to choose a completely new name for this package, rather than `rotary_utils` or `rotary-utils`. That way, we can completely avoid the use of the hyphen/underscore in the package/module names. Unlike with `scikit-learn`, we can't easily truncate `rotary-utils` to something short that is still unique (e.g., `rutils`), and this is another reason why I think it might be better to choose a completely new name. I'll think about some name options and get back to you. How does this sound to you?

For the time being, though, I think `rotary_utils` can be used as a placeholder.

Reply:
@jmtsuji Sounds good. Perhaps a Japanese word as a name?

Reply:
What do you think about `spokewrench`? (Spoke wrenches are used to tune bicycle wheels.) This fits with the `rotary` theme nicely, and there doesn't seem to be another tool in conda with this name.
(I also think it would be nice to use a Japanese name for a tool at some point, but I couldn't think of anything suitable with our current theme this time...)

Reply:
@jmtsuji This sounds good to me. Let's use spokewrench.

Reply:
OK, sounds good!